src/constants/common.ts

```diff
@@ -414,3 +414,8 @@ export const BROWSER_ACTIONS = {
   share: 'share',
   favourite: 'favourite',
 } as const;
+
+export const UPDATE_POINTER_ACTION = {
```

Comment:
Actually this const is not common to every blockchain, it is only for aeternity.